modules/setting/lfs.go

```diff
@@ -20,25 +20,28 @@ var LFS = struct {
 	MaxFileSize     int64         `ini:"LFS_MAX_FILE_SIZE"`
 	LocksPagingNum  int           `ini:"LFS_LOCKS_PAGING_NUM"`
 
-	Storage
+	*Storage
```

Comment:
It needs a explicit name: `Storage *Storage`, then : `lfs.Storage.ServeDirect`

It shouldn't allow developers to write code like `lfs.ServeDirect`, it's quite unclear.